home01/js/script_1.js

```diff
@@ -0,0 +1,5 @@
+// 1. Запросите у пользователя его имя и выведите в ответ: «Привет, его имя!».
+const userName = prompt(`Enter your name:`);
+alert(`Hellow, ${userName}`);
```

Comment:
нужно поговорить с твоим учителем английского 😄 
```suggestion
alert(`Hello, ${userName}`);
```